core/src/main/java/com/digitalpebble/stormcrawler/protocol/okhttp/HttpProtocol.java

```diff
@@ -392,7 +403,7 @@ public ProtocolResponse getProtocolOutput(String url, final Metadata metadata) t
     }
 
     private final byte[] toByteArray(final ResponseBody responseBody,
-            MutableObject trimmed) throws IOException {
+            int maxContent, MutableObject trimmed) throws IOException {
```

Comment:
there is a field _maxContent_ already, which stores the value to be applied in the absence of a per-URL override. To avoid confusion, it would be better to rename that field into something like _globalMaxContent_.